GoRogue.Debugger/Interpreter.cs

```diff
@@ -80,7 +80,10 @@ private static List<IRoutine> GetRoutines()
             // Get all types implementing IRoutine in current assembly
             Type[] routineTypes = Assembly.GetAssembly(typeof(IRoutine))?.GetTypes() ??
                            throw new Exception("Can't find assembly that defines IRoutine.");
-            routineTypes = routineTypes.Where(t => t.GetInterface(nameof(IRoutine)) != null).ToArray();
+            routineTypes = routineTypes
+                .Where(t => t.GetInterface(nameof(IRoutine)) != null)
+                .Where(t => !t.IsAbstract)
```

Comment:
Would it be more performant to do `.Where(t => t.GetInterface(nameof(IRoutine)) != null && !t.IsAbstract)`?

Reply:
Likely yes.  It probably doesn't matter too much here, but the only reason I didn't do it that way to begin with was being lazy about line lengths;  I'll fix this.